File: ruby/zenn_05_02.rb
# 次の多次元配列の
# インデックス0番目のみを取り出した配列を作成して下さい。

array = [
  ["Ruffy", "captain"],
  ["Zoro", "combatant"],
];

ary = array.map{|a| a[0]}
p ary

File: php/zenn_05_02.php
<?php
# 次の多次元配列のインデックス0番目のみを取り出した配列を作成して下さい。
$array = [
  ["Ruffy", "captain"],
  ["Zoro", "combatant"],
];

foreach ($array as $ary) {
  $result[] = $ary[0];
}

print_r($result);
